Navigation.md

```diff
@@ -48,6 +48,7 @@
     * [Create a conversation](/documentation/building-your-bot-on-iadvize#create-a-conversation)
     * [Receive and reply to messages](/documentation/building-your-bot-on-iadvize#receive-a-message-and-reply-to-it)
     * [Conversation payload objects](/documentation/building-your-bot-on-iadvize#conversation-objects)
+  * [Implement bot that initiate conversation](/documentation/building-your-bot-on-iadvize#implement-bot-that-initiate-conversation)
```

Comment:
"Implement a bot that initiates conversations" je dirais plutôt. Mais à valider avec @lforite 

Note : tous mes commentaires en dessous sont liés du coup